serialization/ksp_plugin.proto

```diff
@@ -49,13 +49,20 @@ message Manoeuvre {
 }
 
 message Part {
+  // Pre-Frenet.
+	message InertiaTensor {
```

Comment:
`PreFrenetInertiaTensor`, so that it is clear that this is a compatibility message.